W3ChampionsStatisticService/Services/BackgroundTasks/UpdateMaxMmrService.cs

```diff
@@ -0,0 +1,32 @@
+using System;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.Extensions.Hosting;
+using Microsoft.Extensions.Logging;
+using W3ChampionsStatisticService.Common.Constants;
+using W3ChampionsStatisticService.PlayerProfiles;
+
+namespace W3ChampionsStatisticService.Services.BackgroundTasks;
+public class UpdateMaxMmrService(ILogger<UpdateMaxMmrService> logger, PlayerRepository playerRepository) : BackgroundService
+{
+    private readonly ILogger<UpdateMaxMmrService> _logger = logger;
+    private readonly PlayerRepository _playerRepository = playerRepository;
+
+    protected override async Task ExecuteAsync(CancellationToken stoppingToken)
+    {
+        while (!stoppingToken.IsCancellationRequested)
+        {
+            try
+            {
+                int maxMmr = _playerRepository.LoadMaxMMR();
```

Comment:
Shouldn't a max and min call on an indexed column be a O(1) lookup? Do we need to cache it separately?

Reply:
Okay, then we might at least scope it per game mode and have it both for min and max mmr.